### PR TITLE
Convert AutomatedTests

### PR DESCRIPTION
**Pull request by brianhlin**
https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/AutomatedTests

- [ ] Enter date into "Migrated" column of google sheet
- [ ] Add migration header to TWiki document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +27,107 @@ Other testing frameworks often support fixtures, which bracket a set of tests wi
...
+| `-u`, `--test-user` USERNAME | Use the test user account with the given name. See also the `-a` and `-p` options.                                                                                                                                                         |
+| `-v`, `--verbose`            | Print the name of each test as it is run; generally a good idea.                                                                                                                                                                           |
+| `-h`, `--help`               | Print usage information and exit.                                                                                                                                                                                                          |
+| `--version`                  | Print the script version and exit.                                                                                                                                                                                                         |
+
+### Config file syntax ###
```
Comment (djw8605) [inline README.md]: Does these `###` actually do anything at the end of the line?

File: README.md [hunk 2]
```diff
@@ -24,9 +27,107 @@ Other testing frameworks often support fixtures, which bracket a set of tests wi
...
+| --securepass         | securepass    | False         |
+| --update-release     | updaterelease | None          |
+| --tarballs           | tarballs      | False         |
+| --no-tests           | skiptests     | False         |
+| --test-user          | username      | vdttest       |
+| --verbose            | verbose       | False         |
+|                      | backupmysql   | False         |
+|                      | hostcert      | False         |
+|                      | nightly       | False         |
+|                      | selinux       | False         |
```
Comment (djw8605) [inline README.md]: So there's some osg-test configuration options that do not have command line options?

File: README.md (hunk 1 above)
Comment (brianhlin) [reply README.md]: Nope, it's just what `markdown-mode` does when I create a header < 2. I think it looks pretty :)

File: README.md (hunk 2 above)
Comment (brianhlin) [reply README.md]: Yup, the idea was that we didn't want to expose the more 'advanced' options to users. *shrug*

**Pull request merged**